hikari/api/config.py

```diff
@@ -199,3 +199,13 @@ class CacheSettings(abc.ABC):
     @abc.abstractmethod
     def components(self) -> CacheComponents:
         """Cache components to use."""
+
+    @property
+    @abc.abstractmethod
+    def only_cache_my_member(self) -> bool:
```

Comment:
```suggestion
    def only_my_member(self) -> bool:
```
Maybe better naming? Since everything relates to caching